### DISCUSSION TITLE
Use reserved keywords as form data variable name

### DISCUSSION THREAD

**Discussion by Cleptomania** [fastapi/fastapi#6987] [Questions] [answered]
### First check

* [x] I added a very descriptive title to this issue.
* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.
* [x] I already read and followed all the tutorial in the docs and didn't find an answer.
* [x] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
* [x] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
* [x] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).
* [x] After submitting this, I commit to one of:
    * Read open issues with questions until I find 2 issues where I can help someone and add a comment to help there.
    * I already hit the "watch" button in this repository to receive notifications and I commit to help at least 2 people that ask questions in the future.
    * Implement a Pull Request for a confirmed bug.

### Example

Here's a self-contained, [minimal, reproducible, example](https://stackoverflow.com/help/minimal-reproducible-example) with my use case:

```Python
from fastapi import FastAPI, Form

app = FastAPI()


@app.get("/")
def read_root(from_: str = Form(...)):
    return {"Hello": from_}
```

### Description

In the above example I have a value that comes in as `multipart/form-data` so I am using the Form feature from FastAPI to read it. However, the variable's name coming in is not actually `from_` but rather just `from`. I'm unable to change this as this is being hit from a 3rd party service.

When receiving `application/json` I'm able to get around this by using Pydantic's alias feature. I've tried implementing solutions shown in https://github.com/tiangolo/fastapi/issues/2387. However that doesn't work when the alias names are reserved keywords in python(for example "from" or having a hyphen).

What would be great is if there was a way to alias the Form variables like you can in a Pydantic model using an arbitrary string.

### Environment

* OS: CentOS 7
* FastAPI Version: 0.63.0
* Python version: 3.8.6

**Comment by Kludex** [comment, marked as answer]
`from_: str = Form(..., alias="from")`

**Comment by Cleptomania** [comment]
Thanks I found this just as you commented actually. I will open a pull request to add this to the docs unless you know of it being somewhere already. I wasn't able to find that this was a thing without digging into the source code for `Form`

**Comment by Kludex** [comment]
Not for form specifically, but for `Query`, there is:
![image](https://user-images.githubusercontent.com/7353520/105892648-6f6ca100-6012-11eb-9bf1-2f88ba028a3f.png)

**Comment by tiangolo** [comment]
Thanks for the help here @Kludex ! 👏  🙇 

Thanks for reporting back and closing the issue @Cleptomania  👍

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
